pkg/app/pipedv1/plugin/kubernetes/deployment/misc_test.go

```diff
@@ -277,6 +277,66 @@ spec:
 	}
 }
 
+func TestGenerateVariantWorkloadManifests(t *testing.T) {
+	t.Parallel()
+
+	const (
+		variantLabel  = "pipecd.dev/variant"
+		canaryVariant = "canary-variant"
+	)
+	testcases := []struct {
+		name           string
+		manifestsFile  string
+		configmapsFile string
+		secretsFile    string
+	}{
+		{
+			name:          "No configmap and secret",
+			manifestsFile: "testdata/variant_workload_manifests/no-config-deployments.yaml",
+		},
+		{
+			name:           "Has configmap and secret",
+			manifestsFile:  "testdata/variant_workload_manifests/deployments.yaml",
+			configmapsFile: "testdata/variant_workload_manifests/configmaps.yaml",
+			secretsFile:    "testdata/variant_workload_manifests/secrets.yaml",
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			manifests, err := provider.LoadManifestsFromYAMLFile(tc.manifestsFile)
```

Comment:
Can we add t.Parallel()?

Reply:
Thank you, added https://github.com/pipe-cd/pipecd/pull/5864/commits/7991db8288678c808634ab46fc816c9d6d4d7c85.